pkg/config/manager.go

```diff
@@ -193,7 +193,7 @@ func (c *Manager) fetchAndNotify() {
 
 	glog.Infof("Loaded new config %s", c.store)
 	for _, cl := range c.cl {
-		cl.ConfigChange(rt, df)
+		cl.ConfigChange(rt, df, handlers)
```

Comment:
RT resolves config, which is orthogonal to providing handler instances. I don't think they should be merged ever, really.